modules/components/src/QuickSearch/QuickSearch.js

```diff
@@ -39,39 +39,45 @@ const toggleValue = ({ sqon, setSQON, primaryKeyField, primaryKey }) =>
 	);
 
 const inputRef = React.createRef();
-const QuickSearch = ({
-	apiFetcher,
-	className,
-	documentType,
-	DropdownItemComponent = DropdownItem,
-	headerTitle = 'Search by ID',
-	Icon = FaSearch,
-	InputComponent = TextInput,
-	LoadingIcon = FaSearch,
-	PinnedValueComponent = SQONBubble,
-	placeholder = 'Quick Search',
-	searchLowercase = false,
-	searchTextDelimiters = ['\\s', ','],
-	setSQON,
-	sqon,
-	style = emptyObj,
-	...props
-}) => {
+const QuickSearch = ({ ...props }) => {
```

Comment:
does this affect any other components using the QuickSearch? (not sure how the legacy stuff behaves, as we don't have storybook up and running)